src/AcceptanceTests/NServiceBus.AzureServiceBus.AcceptanceTests.csproj

```diff
@@ -11,12 +11,12 @@
     <AssemblyOriginatorKeyFile>WorkaroundKey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="WindowsAzure.ServiceBus" Version="4.0.0" />
+    <PackageReference Include="WindowsAzure.ServiceBus" Version="4.1.3" />
     <PackageReference Include="Microsoft.WindowsAzure.ConfigurationManager" Version="3.2.3" />
-    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="6.3.1" />
-    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="6.3.1" />
-    <PackageReference Include="NServiceBus" Version="6.3.1" />
-    <PackageReference Include="NUnit" Version="3.6.1" />
+    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="7.0.0-alpharelease0051" />
+    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="7.0.0-alpharelease0051" />
+    <PackageReference Include="NServiceBus" Version="7.0.0-alpharelease0051" />
+    <PackageReference Include="NUnit" Version="3.7.1" />
```

Comment:
Has the issue of having assertions in handlers in ATTs was addressed? NUnit 3.7.1 will work only if assertions are **not** made in handlers (different threads). PR #574 has already attempted that with NSB Core v6 and it didn't work, had to wait for NUnit 3.8.1.

Reply:
I've closed #574 as we no longer need it. This PR essentially replaces it.